<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>353</width>
    <height>261</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="5" column="1">
      <widget class="QLabel" name="markColorLabel">
       <property name="text">
        <string>Pick new mark color:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="pawnColorLabel">
       <property name="text">
        <string>Pick new pawn color:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <layout class="QVBoxLayout" name="highlightBoxLayout">
       <item>
        <widget class="QLabel" name="highlightLabel">
         <property name="text">
          <string>Highlight color:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="highlightColorView">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="boardColorButton">
       <property name="text">
        <string>Open board color picker</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <layout class="QVBoxLayout" name="pawnBoxLayout">
       <item>
        <widget class="QLabel" name="pawnLabel">
         <property name="text">
          <string>Pawn color:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="pawnColorView">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="highlightColorLabel">
       <property name="text">
        <string>Pick new highlight color:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="pawnColorButton">
       <property name="text">
        <string>Open pawn color picker</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <layout class="QVBoxLayout" name="boardBoxLayout">
       <item>
        <widget class="QLabel" name="boardLabel">
         <property name="text">
          <string>Board color:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="boardColorView">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="boardTypeLabel">
       <property name="text">
        <string>Board type:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="highlightColorButton">
       <property name="text">
        <string>Open highlight color picker</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QComboBox" name="comboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <item>
        <property name="text">
         <string>English</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>European</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QPushButton" name="markColorButton">
       <property name="text">
        <string>Open mark color picker</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="boardColorLabel">
       <property name="text">
        <string>Pick new board color:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <layout class="QVBoxLayout" name="markBoxLayout">
       <item>
        <widget class="QLabel" name="markLabel">
         <property name="text">
          <string>Mark color:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="markColorView">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pawnColorButton</tabstop>
  <tabstop>boardColorButton</tabstop>
  <tabstop>highlightColorButton</tabstop>
  <tabstop>markColorButton</tabstop>
  <tabstop>comboBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
